board/body/boards/board_body.h

```diff
@@ -0,0 +1,56 @@
+#include "board/body/boards/motor_control.h"
+
+void board_body_init(void) {
+  motor_init();
+  motor_encoder_init();
+  motor_speed_controller_init();
+  motor_encoder_reset(1);
+  motor_encoder_reset(2);
+}
+
+void board_body_init_bootloader(void) {
+}
+
+void board_body_tick(void) {
+}
```

Comment:
remove all these things that are unused. the board struct isn't the same as panda's so you don't have to populate all these unused fields